src/system_tray.cpp

```diff
@@ -10,10 +10,18 @@
     #include <accctrl.h>
     #include <aclapi.h>
     #define TRAY_ICON WEB_DIR "images/favicon.ico"
-  #elif defined(__linux__) || defined(linux) || defined(__linux)
-    #define TRAY_ICON "sunshine"
```

Comment:
I think on Linux it's required to pull the icons from the `.../share/icons` directory, unless something changed.

`svg` files were used before, as I don't know that `ico` is supported, and they scale better than `png`.